features/netsocket/NetworkInterfaceDefaults.cpp

```diff
@@ -18,7 +18,7 @@
 
 #include "EthInterface.h"
 #include "WiFiInterface.h"
-#include "CellularBase.h"
+#include "CellularInterface.h"
```

Comment:
Can this change be separated from this PR?

Reply:
Sure. No idea why it's even in here.

Reply:
Actually, it already went through as #10146.